jcasc_ouranos/staging_jobs.yaml

```diff
@@ -54,5 +55,44 @@ jobs:
         }
       }
 
+  - script: >
+      pipelineJob('ouranos-staging/test-rook') {
+        description('Test roocs/rook repo. ')
+        definition {
+          cpsScm {
+            scriptPath 'Jenkinsfile'
+            scm {
+              git {
+                remote {
+                  url('https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests.git')
+                }
+                branch('master')
+                extensions {
+                  cleanAfterCheckout()
+                }
+              }
+            }
+            lightweight true
+          }
+        }
+
+        environmentVariables {
+          // This section requires the envinject plugin.
+          // It is known that envinject do not work properly with pipeline.
+          // "These EnvVars are being injected to the script environment and
+          // will be inaccessible via the "env" Pipeline global variable"
+
+          // Can not use DEFAULT_CONFIG_PARAMETERS_SCRIPT_URL because it seems
+          // we can not override global env var.
+          env('DEFAULT_2_CONFIG_PARAMETERS_SCRIPT_URL', 'https://raw.githubusercontent.com/Ouranosinc/PAVICS-e2e-workflow-tests/refs/heads/master/test-override/jenkins-params-external-repos.include.sh')
```

Comment:
I think overrides can be done using `env.DEFAULT_CONFIG_PARAMETERS_SCRIPT_URL = ...` within a `script` definition in Jenkins groovy. Not sure if it can be done here?

Reply:
This section is to specify an env var, it seems like if the env var already existe, it won't override it.  It's a known issue that envinject plugin with pipeline.